fastchat/serve/inference.py

```diff
@@ -498,6 +516,42 @@ def reload_conv(conv):
             conv.messages = new_conv["messages"]
             reload_conv(conv)
             continue
+        elif inp.startswith("!!image-file"):
+            args = inp.split(" ")  # !!image-file <filename> <prompt>
+
+            if len(args) < 3:
+                print("usage: !!image-file <filename> <prompt>")
+                continue
+            else:
+                from PIL import Image
+
+                def load_image(image_file):
```

Comment:
move small utility functions to `fastchat/utils.py` to make major functions shorter